src/libraries/Directory.Build.props

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PortedExtensionsProject Include="$(MSBuildThisFileDirectory)Microsoft.Extensions.Primitives\**\*csproj" />
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)*\tests\**\Microsoft.AspNetCore.Testing\*csproj" Exclude="@(PortedExtensionsProject)" />
```

Comment:
I think this needs a ** after Microsoft.AspNetCore.Testing.  It has to cover test subfolder under that one.